base/v0_5_exp/translate_test.go

```diff
@@ -1699,6 +1699,293 @@ func TestTranslateKernelArguments(t *testing.T) {
 	}
 }
 
+// TestTranslateSSHAuthorizedKeyResource tests translating the butane passwd.users[i].ssh_authorized_keys_local[j] entries to ignition passwd.users[i].ssh_authorized_keys[j] entries.
+func TestTranslateSSHAuthorizedKeyResource(t *testing.T) {
+	sshKeyDir := t.TempDir()
+	randomDir := t.TempDir()
+	var sshKey = "ssh-rsa AAAAAAAAA"
+	var sshKeyFileName = "id_rsa.pub"
+	var sshKeyMultipleKeysFileName = "multiple.pubs"
+	var sshKeyNonExistingFileName = "id_ed25519.pub"
+	var sshKeyEmptyFileName = "empty.pub"
+	var sshKeyBlankFileName = "blank.pub"
+	err := ioutil.WriteFile(filepath.Join(sshKeyDir, sshKeyFileName), []byte(sshKey), 0644)
+	if err != nil {
+		t.Error(err)
+	}
+	err = ioutil.WriteFile(filepath.Join(sshKeyDir, sshKeyMultipleKeysFileName), []byte(fmt.Sprintf("%s\n%s\n\n", sshKey, sshKey)), 0644)
+	if err != nil {
+		t.Error(err)
+	}
+	err = ioutil.WriteFile(filepath.Join(sshKeyDir, sshKeyEmptyFileName), []byte(""), 0644)
+	if err != nil {
+		t.Error(err)
+	}
+	err = ioutil.WriteFile(filepath.Join(sshKeyDir, sshKeyBlankFileName), []byte("\n\t\n\n\n\n\n\n"), 0644)
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
Optional: this could be a loop over a `map[string][]byte` mapping filename -> contents.